packages/fether-react/src/contexts/AccountContext.js

```diff
@@ -0,0 +1,11 @@
+// Copyright 2015-2018 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+//
+// SPDX-License-Identifier: BSD-3-Clause
+
+import { withProps } from 'recompose';
+
+// react-router doesn't pass match params to child components
+export const consumeAccount = withProps(({ match: { url } }) => ({
+  accountAddress: /^\/?tokens\/([^/]+)/.exec(url)[1]
```

Comment:
I think in the match object, there's a `params` field, so that accountAddress is matche.params.accountAddress.

Reply:
The params object seems to exist for the component specified in the `<Route>`, but not for the nested components, even though they have `withRouter`

Reply:
Ah, that's too bad.